src/renderers/shared/shared/event/BrowserEventConstants.js

```diff
@@ -64,6 +64,8 @@ var topLevelTypes = {
   topLoadedMetadata: 'loadedmetadata',
   topLoadStart: 'loadstart',
   topMouseDown: 'mousedown',
+  topMouseEnter: 'mouseenter',
+  topMouseLeave: 'mouseleave',
```

Comment:
I wonder how irregular the mapping is, could maybe remove it in favor of something like we do here: https://github.com/jquense/react/blob/2a69ba16caf4eee8c9f1dcfbeb937049c494cec3/src/renderers/dom/shared/eventPlugins/SimpleEventPlugin.js#L133-L135